Makefile

```diff
@@ -52,4 +53,5 @@ docker-all:
 
 .PHONY: docker-bash
 docker-bash:
-	-docker-compose run --rm dbmate bash
+	docker-compose build
+	docker-compose run --rm dbmate bash
```

Comment:
can you revert this bit

Reply:
Reverted. Sorry, it must be an artefact of a recent rebase.